keps/sig-node/runtime-class.md

```diff
@@ -270,6 +271,34 @@ the Pod. The admission check on a replica recreation will prevent the scheduler
 the `RuntimeHandler` is not recognized by the CRI implementation, then `RunPodSandbox` will return
 an error.
 
+#### Monitoring
+
+The first round of monitoring implementation for `RuntimeClass` will cover the
+following three areas:
+
+- `how popular every runtime is?` This can be done by adding a new label into
+  [kubelet\_running\_pod\_count](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/metrics/metrics.go#L256)
```

Comment:
nit: Links to specific lines should always pin a commit, otherwise the line may drift. Just press 'y' on github to replace the URL with a pinned version.